packages/react/presence/package.json

```diff
@@ -21,7 +21,8 @@
     "@radix-ui/react-use-layout-effect": "workspace:*"
   },
   "peerDependencies": {
-    "react": "^16.8 || ^17.0 || ^18.0"
+    "react": "^16.8 || ^17.0 || ^18.0",
+    "react-dom": "^16.8 || ^17.0 || ^18.0"
```

Comment:
Because of this, we will need to track all of the dependents of `Presence` and their dependends subsequently and add that peer-dependency too.

Sometimes I wonder if it would be easier to always add them both as peer-dep anyway haha, would there be any harm considering that they are meant to be used with the DOM anyway?

Reply:
Good point!

> Sometimes I wonder if it would be easier to always add them both as peer-dep anyway haha, would there be any harm considering that they are meant to be used with the DOM anyway?

Practically speaking I don't think it'd be a problem, technically inaccurate in terms of the dependency contracts but maybe that's ok in this case? at least we'd never forget in the future.

Reply:
Yeah exactly, I'm mostly worried that it's hard to track through multiple levels of dependencies, if it was just flat, it might be ok, but it becomes a bit harder when there's more than 1 level.